doc/user/evpn.rst

```diff
@@ -502,3 +502,26 @@ the creation of the VLAN subinterfaces and directly enslave "eth0" to "br10".
       ip link set eth0 master br10
 
 This completes the necessary configuration for an L2VNI.
+
+Displaying EVPN Information
+----------------------------------------
+
+.. clicmd:: show vrf [<NAME$vrf_name|all$vrf_all>] vni [json]
+
+   Displays VRF to L3VNI mapping. It also displays L3VNI associated
+   router-mac, svi interface and vxlan interface.
+   User can get that information as JSON format when ``json`` keyword
+   at the end of cli is presented.
+
+   .. code-block:: frr
+
+      tor2# show vrf vni
+      VRF                                   VNI        VxLAN IF             L3-SVI               State Rmac
+
```

Comment:
This new line looks odd (after the header). The old output does not have this extra newline or this is just an error in copy/paste?

Reply:
No extra line in actual output
```
TORC11# show vrf vni
VRF                                   VNI        VxLAN IF             L3-SVI               State Rmac
vrf1                                  4001       vx-4001              vlan4001             Up    44:38:39:ff:ff:01
vrf2                                  4002       vx-4002              vlan4002             Up    44:38:39:ff:ff:01
vrf3                                  4003       vx-4003              vlan4003             Up    44:38:39:ff:ff:01
```